packages/apollos-data-connector-rock/src/auth-sms/data-source.js

```diff
@@ -169,7 +169,9 @@ export default class AuthSmsDataSource extends RockApolloDataSource {
 
     await this.context.dataSources.Sms.sendSms({
       to: e164,
-      body: `Your login code is ${pin}`,
+      body: `${
+        process.env.SMS_LOGIN_MESSAGE || 'Thanks for logging in!'
```

Comment:
```suggestion
        ApollosConfig.SMS_LOGIN_MESSAGE || 'Thanks for logging in!'
```

I don't think we need to touch the template, since we have a default.

Reply:
Then at the top of the file, 
```
import ApollosConfig from '@apollosproject/config';
```